examples/bioasq9b.py

```diff
@@ -0,0 +1,232 @@
+"""
+BioASQ Task B On Biomedical Semantic QA (Involves IR, QA, Summarization qnd
+More). This task uses benchmark datasets containing development and test
+questions, in English, alongwith gold standard (reference) answers constructed
+by a team of biomedical experts. The participants have to respond with relevant
+concepts, articles, snippets and RDF triples, fromdesignated resources, as well
+as exact and 'ideal' answers.
+
+Fore more information about the challenge, the organisers and the relevant
+publications please visit: http://bioasq.org/
+"""
+import json
+import os
+import glob
+import datasets
+
+
+_CITATION = """\
+@article{tsatsaronis2015overview,
+	title        = {
+		An overview of the BIOASQ large-scale biomedical semantic indexing and
+		question answering competition
+	},
+	author       = {
+		Tsatsaronis, George and Balikas, Georgios and Malakasiotis, Prodromos and
+		Partalas, Ioannis and Zschunke, Matthias and Alvers, Michael R and
+		Weissenborn, Dirk and Krithara, Anastasia and Petridis, Sergios and
+		Polychronopoulos, Dimitris and others
+	},
+	year         = 2015,
+	journal      = {BMC bioinformatics},
+	publisher    = {BioMed Central Ltd},
+	volume       = 16,
+	number       = 1,
+	pages        = 138
+}
+"""
+
+_DESCRIPTION = """\
+The data are intended to be used as training and development data for BioASQ 9, which will
+take place during 2021. There is one file containing the data:
+ - training9b.json
+
+The file contains the data of the first seven editions of the challenge: 3742 questions [1]
+with their relevant documents, snippets, concepts and RDF triples, exact and ideal answers.
+For more information about the format of the data as well as the instructions for
+participating at BioASQ please consult: 
+http://participants-area.bioasq.org/general_information/Task9b/
+
+Differences with BioASQ-training8b.json 
+	- 499 new questions added from BioASQ8
+		- The question with id 5e30e689fbd6abf43b00003a had identical body with 
+          5880e417713cbdfd3d000001. All relevant elements from both questions are available
+          in the merged question with id 5880e417713cbdfd3d000001.
+"""
+
+_HOMEPAGE = "http://participants-area.bioasq.org/datasets/"
+
+# Data access reqires registering with BioASQ.
+# See http://participants-area.bioasq.org/accounts/register/
+_LICENSE = ""
+
+_URLs = {"train": "BioASQ-training9b.zip", "test": "Task9BGoldenEnriched.zip"}
+
+
+class Bioasq9bDataset(datasets.GeneratorBasedBuilder):
+    """BioASQ Task B On Biomedical Semantic QA."""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="original",
+            version=VERSION,
+            description="Original BioASQ9 source schema.",
+        ),
+        datasets.BuilderConfig(
+            name="bigbio",
+            version=VERSION,
+            description="Simplified schema for BigScience-Biomedical QA tasks",
+        ),
+    ]
```

Comment:
I like the `original` and `bigbio` config names ... we might want to do something like `bigbio-schema` (e.g. bigbio-kb or bigbio-text-pairs) as some datasets support multiple tasks. Specifically I'm thinking of things like the MuchMore dataset which can be a translation dataset or an NER dataset. `bigbio-<schema>` instead of `bigbio-task` or just `bigbio` handles this well i think